mcstatus/server.py

```diff
@@ -150,25 +150,33 @@ async def _retry_async_status(self, connection: TCPAsyncSocketConnection, **kwar
         result = await pinger.read_status()
         return result
 
-    def query(self) -> QueryResponse:
-        """Checks the status of a Minecraft Java Edition server via the query protocol."""
+    def query(self, tries: int = 3) -> QueryResponse:
```

Comment:
Actually, there is one nitpick I'd make here, should `tries`  be keyword only argument here too?
Doing `JavaServer.query(1)` looks a bit weird to me as it's pretty unclear what that 1 is for.

```suggestion
    def query(self, *, tries: int = 3) -> QueryResponse:
```

Same remark for `async_query`